src/classes/RD2_ElevateInformation_CTRL.cls

```diff
@@ -59,9 +59,15 @@ public with sharing class RD2_ElevateInformation_CTRL {
     public static Map<String, Object> getData(Id recordId) {
         Map<String, Object> data = new Map<String, Object>();
 
+        Boolean hasRDUpdatePermissions = UTIL_Permissions.canUpdate('npe03__Recurring_Donation__c', false)
+            && UTIL_Permissions.canUpdate('npe03__Recurring_Donation__c', String.valueOf(npe03__Recurring_Donation__c.CardLast4__c), false)
+            && UTIL_Permissions.canUpdate('npe03__Recurring_Donation__c', String.valueOf(npe03__Recurring_Donation__c.CardExpirationMonth__c), false)
```

Comment:
@andrewyu-salesforce This may be a direct conflict with the permissions checks I introduce in #6242. In that PR to show the credit card last 4 and exp. date, it does not require perms to these fields, but instead only displays the information on the UI if they have perms to these fields.  I suspect that this addition will cause that logic to stop working, and instead never render the card info unless the user can update these fields (not just read them). We may need to reconcile that difference before one or both of the PR's are tested/merged.

Reply:
The class is now update to reflect PR #6242 and pass 2 variables to LWC. If the user does not have update permission, then they cannot see the `Update Payment Information` button, and in #6242 will take care of the displaying whole component based on the read access.